core/src/main/web/app/helpers/core.js

```diff
@@ -683,16 +682,26 @@
         if (!msgHeader) {
           msgHeader = "Oops...";
         }
-        btnText = btnText ? btnText : "Close";
-        btnClass = btnClass ? _.isArray(btnClass) ? btnClass.join(' ') : btnClass : 'btn-primary';
-        var template = "<div class='modal-header'>" +
-            "<h1>" + msgHeader + "</h1>" +
-            "</div>" +
-            "<div class='modal-body'><p>" + msgBody + "</p></div>" +
-            '<div class="modal-footer">' +
-            "   <button class='btn " + btnClass +"' ng-click='close(\"OK\")'>" + btnText + "</button>" +
-            "</div>";
-        return this.showModalDialog(callback, template);
+        if (bkUtils.isElectron) {
+          var options = {
+            type: 'none',
+            buttons: ['OK'],
+            title: msgHeader,
+            message: msgBody
+          };
+          return bkUtils.Electron.Dialog.showMessageBox(options, callback);
+        } else {
+          btnText = btnText ? btnText : "Close";
+          btnClass = btnClass ? _.isArray(btnClass) ? btnClass.join(' ') : btnClass : 'btn-primary';
+          var template = "<div class='modal-header'>" +
```

Comment:
this modal code was already super copy and pasted before this, lets see if we can DRY this code up.

the if / else blocks in each of these message blocks for env specific display looks like it is begging for some abstaction.